### PR TITLE
Mm 02 create rover selector component

### PR DESCRIPTION
**Pull request by JackHughes90**
- Created route to rover selector
- Created Rover component

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -3,13 +3,16 @@ import "./App.scss";
 import { Header } from "./navigation/Header";
 import { Route, Routes } from "react-router-dom";
 import { LandingPage } from "./landingPage/LandingPage";
+import { RoverSelectorPage } from "./roverSelector/RoverSelectorPage";
+
```
Comment (TimLeach635) [inline src/components/App.tsx:7]: Extra blank line! Just one blank line separating these two please 😊

File: src/components/roverSelector/RoverSelectorPage.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export const RoverSelectorPage: React.FunctionComponent = () => {
+    return (
+      <main>
+        hello
+      </main>
+    );
+  };
```
Comment (TimLeach635) [inline src/components/roverSelector/RoverSelectorPage.tsx]: See how GitHub has added a little ⛔ at the bottom of the file here? This is to indicate that the file doesn't end with a single empty line, which is usually preferred. Could you add one here please?

Comment (TimLeach635) [inline src/components/roverSelector/RoverSelectorPage.tsx]: I think all these lines are over-indented by one level!

**Review by TimLeach635 — CHANGES_REQUESTED** [review]
Looking great! Just some cosmetic/style notes

File: src/components/App.tsx (hunk 1 above)
Comment (JackHughes90) [reply src/components/App.tsx:7]: Extra line removed

File: src/components/roverSelector/RoverSelectorPage.tsx (hunk 2 above)
Comment (JackHughes90) [reply src/components/roverSelector/RoverSelectorPage.tsx]: Extra line added

**Review by TimLeach635 — APPROVED** [review]
Perfect!

**Pull request merged**